src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -359,9 +352,6 @@ def from_dict(cls, config_dict):
 
         return ens_config
 
-    def __len__(self):
-        return self._size()
-
```

Comment:
i was surprised by this deletion, because i remembered that ensemble size was something used here and there, in particular in libres facade. checking the code, in enkf main, we find

```
self._ensemble_size = self.res_config.model_config.num_realizations
```
it obviously has nothing to do with ensemble config, how silly of me :smiling_face_with_tear: